Mobile/starter_project_matador/lib/features/example/presentation/screen/onboarding_1_screen.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:matador/features/example/presentation/widgets/animated_entry.dart';
+import 'package:matador/features/example/presentation/widgets/introduction.dart';
+
+class OnBoarding1 extends StatefulWidget {
+  const OnBoarding1({super.key});
+
+  @override
+  State<OnBoarding1> createState() => _OnBoardingState();
+}
+
+class _OnBoardingState extends State<OnBoarding1> {
+  double displaced = -350;
+  String topic = "Read the article you want instantly";
+  String content =
+      "You can read thousands of articles on Blog Club, save them in the application and share them with your loved ones.";
+  
+  @override
+  void initState() {
+    super.initState();
+    Future.delayed(const Duration(milliseconds: 500), () {
+      setState(() {
+        displaced = 0;
+      });
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var size = MediaQuery.of(context).size;
+    return Column(children: [
+      Container(
+        margin: EdgeInsets.only(top: size.height * 0.01),
+        padding: const EdgeInsets.all(10),
+        width: size.width,
+        height: size.height * (2.8 / 5),
+        child: Stack(
+          children: [
+            AnimatedEntry(
+              width: size.width * (1 / 3) * 0.9,
+              height: size.height * (3 / 5) * 0.4,
+              image: const AssetImage("assets/images/background1.png"),
```

Comment:
my above comment applies here too
if possible i would recommend not numbering things but describing them in their names

Reply:
[nit] i am not sure if it is an option but is there a way to add the AnimatedEntrys in a loop? (they look to me like they have a lot of common parts)
idk maybe it is weird to handle the right, top, left.. entries tho

Reply:
Thanks for your comments
- There is a second onboarding page that I am currently working on and that is why it is named like that.
- You are right on the naming the numbers and I will fix that.
- I don't think adding the AnimatedEntries in a loop is handy, because each of them have different properties and there is no pattern between them.